mitodl/ocw-hugo-themes
# pass history to course-search-utils

#### Pre-Flight checklist


#### What are the relevant tickets?
Accounts for a breaking change introduced in https://github.com/mitodl/course-search-utils/pull/36

#### What's this PR do?
Updates SearchPage to account for a breaking change in course-search-utils. course-search-utils now requires passing a history object.

#### How should this be manually tested?

1. Check out this branch
2. In a local copy of [course-search-utils](https://github.com/mitodl/course-search-utils)
    ```sh
    git checkout cc/history
    yarn pack
    # move the tgz file to your ocw-hugo-themes repo. If the repos as siblings, then:
    cp mitodl-course-search-utils-v1.8.0.tgz ../ocw-hugo-themes/mitodl-course-search-utils-v1.8.0.tgz
    ```
3. In the ocw-hugo-themes repo:
    ```sh
    yarn install
    yarn run start:www
    ```
4. Check that search page still updates the URL bar appropriately.
